assets/common/models.go

```diff
@@ -15,6 +15,7 @@ type ConcourseParams struct {
 	//postMessage Params
 	Channel         string `json:"channel"`
 	AttachmentsFile string `json:"attachments_file"`
+	Attachments     string `json:"attachments"`
```

Comment:
Just a thought although it would be a breaking change. Would it make sense to provide a single parameter, say 'Attachments', and if a file interpret it as such; if valid JSON interpret it as such; otherwise an error?

Reply:
I'd prefer not to.  The other resources I've seen use this paradigm and handling the breaking change would be a bummer.  That cool if we don't?

Reply:
cool